src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -615,7 +631,16 @@ export class RoslynLanguageServer {
             childProcess = cp.spawn(serverPath, args, cpOptions);
         }
 
-        return childProcess;
+        //Log any stderr/stdout messages in our output channel.
+        childProcess.stderr.on('data', (data: { toString: (arg0: any) => any }) =>
```

Comment:
At this point, it should basically just be the console logger (before LSP is setup) that we log right?  Did you see anything else in there?

Potentially as well we should prefix these logs with something like [stdout] or [stderr] to indicate where the log is coming from?

Reply:
The only thing I see is the console logger and the output from Unity.  I figured there wasn't a downside to including it, expect maybe its confusing where all of this is coming from? I think adding a prefix is a good compromise. Are we good with that?

Reply:
Yeah I agree we should leave it in (with a prefix).